### DISCUSSION TITLE
Frigate Docker image under a Linux Live distro

### DISCUSSION THREAD

**Discussion by sudo-joe** [blakeblackshear/frigate#3791] [General]
Hello forum

would like to test Frigate. Will this Frigate Docker image run properly under a Linux Live distro (started from USB) like Ubuntu or Linux Mint?

Thank you!

Joe

**Comment by NickM-27** [comment]
I don't see why it wouldn't. Many users (including myself) run unRAID OS with frigate in docker, unRAID runs in this way.
